sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 1.16.1 (2024-05-24)
```

Comment:
Reminder for updating `_version.py`

https://msdata.visualstudio.com/Vienna/_git/sdk-cli-v2?path=/docs/azure-sdk-for-python/release_and_hotfix_process.md&_a=preview&anchor=hotfix-process

Reply:
Thanks Billy!